generators/server/templates/package.json.ejs

```diff
@@ -36,9 +37,8 @@
     <%_ } _%>
     "prettier": "<%= PRETTIER_VERSION %>",
```

Comment:
There will be an issue if `prettierJava` is `false`, because of the trailing comma here.

Reply:
oh right, nice catch ! I'm fixing it :D